code/game/objects/items/storage/pouch.dm

```diff
@@ -154,7 +154,7 @@
 	new /obj/item/reagent_containers/hypospray/autoinjector/combat (src)
 	new /obj/item/reagent_containers/hypospray/autoinjector/combat (src)
 	new /obj/item/reagent_containers/hypospray/autoinjector/oxycodone (src)
-	new /obj/item/reagent_containers/hypospray/autoinjector/synaptizine_expired (src)
+	new /obj/item/reagent_containers/hypospray/autoinjector/russian_red (src)
```

Comment:
this smells like a recipe for disaster

Reply:
Russian Red buffed significantly, to fill the T1 emergency healchem role better, and maybe be actually usable.
Now restores 50 stamina on mob_add. Heals 100 brute and burn and 25 toxin over 10 ticks, up from 50 brute and burn. Cloneloss dropped from 20 per 10u to 5 per 10u. 10 per 10u.

Reply:
not a justification